index.d.ts

```diff
@@ -1,20 +1,12 @@
 import { ChildProcess } from 'child_process';
 
-declare module 'dynamodb-local' {
-  type argValues = '-cors' | '-dbPath' | '-delayTransientStatuses' | '-help' | '-inMemory' | '-optimizeDbBeforeStartup' | '-port' | '-sharedDb';
+type argValues = '-cors' | '-dbPath' | '-delayTransientStatuses' | '-help' | '-inMemory' | '-optimizeDbBeforeStartup' | '-port' | '-sharedDb';
 
-  export interface InstallerConfig {
-    installPath: string;
-    downloadUrl: string;
-  }
-
-  namespace DynamoDbLocal {}
-
-  export class DynamoDbLocal {
-    static configureInstaller(config: InstallerConfig): void;
-    static launch(portNumber: number, dbPath: string | null, args: argValues[], verbose?: boolean): Promise<ChildProcess>;
-    static stop(portNumber: number): void;
-  }
-
-  export = DynamoDbLocal;
+export interface InstallerConfig {
+  installPath: string;
+  downloadUrl: string;
 }
+
+export function configureInstaller(config: InstallerConfig): void;
+export function launch(portNumber: number, dbPath?: string | null, args?: argValues[], verbose?: boolean): Promise<ChildProcess>;
```

Comment:
Only change, other than removing the wrapper: marked `dbPath` and `args` as optional.  (Otherwise TS insists that you do `launch(PORT, null, [])`, rather than just `launch(PORT)`.